### PR TITLE
Device: JM Speech Synthesis v3.3

### PR DESCRIPTION
**Pull request by jamesadevine**
This pull request registers a new device for Jacdac.

### CODE REVIEW

**Comment by jamesadevine** [comment]
The form didn't allow me to specific a design id or revision on the registration page. The image also looked weird when it was rendered on the registration page.

I'll reupload image here:
IMG_9449

File: devices/microsoft/research/jmspeechsynthesisv33.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "description": "JM Speech Synthesis v3.3",
+  "repo": "https://github.com/microsoft/jacdac-msr-modules",
+  "services": [
+    302307733
+  ],
+  "productIdentifiers": [
+    934541191
+  ],
+  "version": "l",
+  "designIdentifier": "1"
```
Comment (pelikhan) [inline devices/microsoft/research/jmspeechsynthesisv33.json:14]: Can you use server? 1.0

**Review by pelikhan — APPROVED** [review]

**Review by pelikhan — APPROVED** [review]
I need to fix my rescaling logic.... post the original pic on the bug

File: devices/microsoft/research/jmspeechsynthesisv33.json (hunk 1 above)
Comment (jamesadevine) [reply devices/microsoft/research/jmspeechsynthesisv33.json:14]: I didn't enter these fields, as described above

Comment (jamesadevine) [reply devices/microsoft/research/jmspeechsynthesisv33.json:14]: Correct revision is 3.3, no design identifier has been assigned to this proto board.

Comment (jamesadevine) [reply devices/microsoft/research/jmspeechsynthesisv33.json:14]: we could probably use design id 61 as it is a previous version of the latest mikrobus module

**Pull request merged**